services/cd-service/pkg/grpc/errors.go

```diff
@@ -37,11 +37,15 @@ func PublicError(ctx context.Context, err error) error {
 	return status.Error(codes.InvalidArgument, "error: "+err.Error())
 }
 
-func CanceledError(ctx context.Context, err error) error {
+func CanceledError(_ context.Context, err error) error {
 	return status.Error(codes.Canceled, err.Error())
 }
 
-func AuthError(ctx context.Context, err error) error {
+func FailedPrecondition(_ context.Context, err error) error {
+	return status.Error(codes.FailedPrecondition, err.Error())
+}
+
+func AuthError(_ context.Context, err error) error {
 	return status.Error(codes.Unauthenticated, "error: "+err.Error())
```

Comment:
inconsistent with line 45/41, can we fix that?

Reply:
done